docs/pc-widget-unity/configuration.mdx

```diff
@@ -0,0 +1,147 @@
+---
+sidebar_position: 50
+title: Configuration
+description: "Configuration options for Helpshift PC widget - Unity"
+---
+
+import {
+  Image
+} from "@site/src/components/forDocs";
+
+
+## Configuration
+
+A configuration object has to be created, which should be passed to the `initialize` API.
+Below are the configuration options that we have.
+
+### Properties:
```

Comment:
Use table.

Required should be also part of the table, with true & false info. True -> config is required,